pkg/api/api.go

```diff
@@ -256,6 +256,7 @@ func Register(r *macaron.Macaron) {
 		r.Group("/alerts", func() {
 			r.Post("/test", bind(dtos.AlertTestCommand{}), wrap(AlertTest))
 			r.Post("/:alertId/pause", bind(dtos.PauseAlertCommand{}), wrap(PauseAlert), reqEditorRole)
+			r.Post("/pause", bind(dtos.PauseAlertsCommand{}), wrap(PauseAlerts), reqGrafanaAdmin)
```

Comment:
if's all alerts in the instance in needs to be a different HTTP api endpoint, /api/admin/  or something because /api/alerts only affects the current org .

And since we need a UI button for this in the alerts page I think making only act on the org alerts makes the most sense for now. I think the action for "ALL" alerts across multiple orgs does not make much sense, for that case I think we can have a "pause alert scheduler" or something :)